enterprise/models.py

```diff
@@ -121,6 +121,13 @@ class Meta:
         )
     )
 
+    hide_audit_track = models.BooleanField(
```

Comment:
Could we change this to `enable_audit_enrollment` with a default of `False`? To me, this is a better description of what this field means and is a more appropriate default based on the use cases I have heard.